src/components/index.js

```diff
@@ -1,5 +1,6 @@
 export * from "./avatar";
 export * from "./button";
+export * from "./camera";
```

Comment:
You'll also need an `index.js` file inside the `/camera` directory-- look at some of the other components to see how we do the exports. Once you do this, you should be able to do `import { Camera } from "../components"` in files like your routes.